src/Field.js

```diff
@@ -0,0 +1,18 @@
+import {useState} from 'react'
+
+function Field({label, className, type, ref, onChange}) {
+    const [showPassword, setShowPassword] = useState(false);
+  
+    const handlePasswordVisibility = ()=>{
+      setShowPassword(!showPassword)
+    }
+    return (
+      <div className={`flex__column ${className}`}>
+      {type === 'password' ? <div onClick={handlePasswordVisibility} className='form__password__visibility__handler'>OKO</div> : null }
```

Comment:
This is fine in this exercise. However you can probably see that we are introducing a lot of logic related to just password. It's a perfect case to extract that logic into something like `FieldPassword`.